gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/parameters/Key.java

```diff
@@ -319,7 +319,9 @@ public enum Key {
     JUPITER_MODE_ENABLED("api.jupiterMode.enabled", "false", Set.of(SYSTEM)),
 
     // TODO: switch to 'always' to enabled Jupiter execution mode by default when jupiter will go GA.
-    JUPITER_MODE_DEFAULT("api.jupiterMode.default", "never", Set.of(SYSTEM));
+    JUPITER_MODE_DEFAULT("api.jupiterMode.default", "never", Set.of(SYSTEM)),
+
+    ALERT_ENGINE_ENABLED("alerts.alert-engine.enabled", "false", new HashSet<>(Arrays.asList(ORGANIZATION)));
```

Comment:
Would that be more relevant to have SYSTEM instead of ORGANIZATION ? (Just asking, I don't know the difference)

Reply:
And can't we use  `Set.of` ?

Reply:
done